Adafruit_IO/client.py

```diff
@@ -22,13 +22,18 @@
 import json
 import platform
 import pkg_resources
+import re
+from urllib.parse import urlparse
+from urllib.parse import parse_qs
 # import logging
 
 import requests
 
 from .errors import RequestError, ThrottlingError
 from .model import Data, Feed, Group
 
+API_PAGE_LIMIT = 1000
```

Comment:
Could you comment what this is?

Reply:
Currently, the web API returns a maximum of 1000 data points per [page](https://io.adafruit.com/api/docs/#pagination). I could have hard-coded this into the function definition for `data`, but I prefer to have constants like this defined up front. That way if the API were to change for whatever reason, the python library would only need to be updated in this one location.